.golangci.yml

```diff
@@ -0,0 +1,18 @@
+linters-settings:
+  misspell:
+    locale: US
+
+linters:
+    enable:
```

Comment:
all those linters are enabled by default and there are more enabled linters!
if you want to control which are enabled, consider the following example from the docs https://github.com/golangci/golangci-lint#config-file:
```
linters:
  enable:
    - megacheck
    - govet
  enable-all: false
  disable:
    - maligned
    - prealloc
  disable-all: false
  presets:
    - bugs
    - unused
  fast: false
```
I think we should lock down the linters that way

Reply:
Locking down the linters like that looks nice though, I'll investigate that.